system/View/View.php

```diff
@@ -493,20 +493,21 @@ public function endSection()
 	 *
 	 * @param string $sectionName
 	 */
-	public function renderSection(string $sectionName)
+	public function renderSection(string $sectionName) : string
```

Comment:
while in documentation, the `renderSection()` is called as : 

```php
    <?= $this->renderSection('content') ?>
```

It may cause a bc break to user when user just use as is:

```php
 <?php $this->renderSection('content') ?>
```

It may require a 2nd parameter `$echoed` = true I think. //cc @natanfelles

Reply:
I just added 2nd parameter `$echoed` default to true to avoid bc break.